Sources/Support/PurchasesDiagnostics.swift

```diff
@@ -132,6 +132,16 @@ extension PurchasesDiagnostics {
         /// Failure performing a signed request
         case failedMakingSignedRequest(Swift.Error)
 
+        /// Any other not identifier error. You can check the undelying error for details.
+        case unknown(Swift.Error)
+
+    }
+
+    /// An error that represents a problem in the SDK's configuration
+    public enum SDKHealthError: Swift.Error {
```

Comment:
Could this type be added under the `DEBUG` compiler flag?

Reply:
I think so, I will move all of the health report types and entities under the `DEBUG` compiler flag

Reply:
Updated!